zipline/finance/performance/tracker.py

```diff
@@ -423,7 +427,7 @@ def handle_market_close_daily(self):
         rate.
         """
         self.update_performance()
-        completed_date = normalize_date(self.market_close)
+        completed_date = normalize_date(self.day)
```

Comment:
It looks like your change means that `self.day` is always midnight.
So this call to `normalize_date` may now be redundant.